<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogSystemPluginParameter</class>
 <widget class="QDialog" name="DialogSystemPluginParameter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>637</width>
    <height>412</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <property name="leftMargin">
    <number>15</number>
   </property>
   <property name="topMargin">
    <number>10</number>
   </property>
   <property name="rightMargin">
    <number>15</number>
   </property>
   <property name="bottomMargin">
    <number>10</number>
   </property>
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,0">
     <property name="spacing">
      <number>15</number>
     </property>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,1">
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="labelName">
           <property name="text">
            <string>Name:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,1">
           <property name="spacing">
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="labelNameValue">
             <property name="text">
              <string>Name</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEditName"/>
           </item>
          </layout>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="labelDescription">
           <property name="text">
            <string>Description:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,1">
           <property name="spacing">
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="labelDescriptionValue">
             <property name="text">
              <string>Description</string>
             </property>
             <property name="wordWrap">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEditDescription"/>
           </item>
          </layout>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="labelDefaultValue">
           <property name="text">
            <string>Default value:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="PanelPrimitiveValue" name="panelPrimitiveDefaultValue" native="true">
           <property name="minimumSize">
            <size>
             <width>500</width>
             <height>27</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>500</width>
             <height>27</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayoutVectors">
         <item row="3" column="0">
          <widget class="QCheckBox" name="checkBoxMin">
           <property name="text">
            <string>Min:</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QCheckBox" name="checkBoxMax">
           <property name="text">
            <string>Max:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="labelZ">
           <property name="text">
            <string>Z:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="0,1">
           <item>
            <widget class="PanelPrimitiveValue" name="panelPrimitiveValueZ" native="true">
             <property name="minimumSize">
              <size>
               <width>250</width>
               <height>27</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>250</width>
               <height>27</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_3">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>0</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item row="1" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="0,1">
           <item>
            <widget class="PanelPrimitiveValue" name="panelPrimitiveValueY" native="true">
             <property name="minimumSize">
              <size>
               <width>250</width>
               <height>27</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>250</width>
               <height>27</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>0</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="labelX">
           <property name="text">
            <string>X:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,1">
           <item>
            <widget class="PanelPrimitiveValue" name="panelPrimitiveValueX" native="true">
             <property name="minimumSize">
              <size>
               <width>250</width>
               <height>27</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>250</width>
               <height>27</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>0</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="labelY">
           <property name="text">
            <string>Y:</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="0,1">
           <item>
            <widget class="QSpinBox" name="spinBoxMax">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="minimumSize">
              <size>
               <width>100</width>
               <height>0</height>
              </size>
             </property>
             <property name="minimum">
              <number>-16777215</number>
             </property>
             <property name="maximum">
              <number>16777215</number>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_4">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>0</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item row="3" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout_7" stretch="0,1">
           <item>
            <widget class="QSpinBox" name="spinBoxMin">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="minimumSize">
              <size>
               <width>100</width>
               <height>0</height>
              </size>
             </property>
             <property name="minimum">
              <number>-16777215</number>
             </property>
             <property name="maximum">
              <number>16777215</number>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_5">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>0</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="WidgetTreeStructure" name="treeView">
         <attribute name="headerVisible">
          <bool>false</bool>
         </attribute>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PanelPrimitiveValue</class>
   <extends>QWidget</extends>
   <header>panelprimitivevalue.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>WidgetTreeStructure</class>
   <extends>QTreeView</extends>
   <header>widgettreestructure.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogSystemPluginParameter</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogSystemPluginParameter</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
